server/routes/v1/user.ts

```diff
@@ -0,0 +1,42 @@
+import express, { Request, Response } from "express";
+import db from "../../models";
+import EmailService from "../../services/emailService";
+import UserController from "../../controllers/v1/user";
+
+const userRouter = express.Router();
+const eServ = new EmailService();
+const uContr = new UserController(db.User, eServ);
+
+userRouter.get(
+  "/confirmation/c=:token",
+  async (req: Request, res: Response) => {
+    const status: boolean = await uContr.confirmEmail(req.params.token);
+
+    if (status) {
+      res.status(200).send("Your email address has been confirmed. ");
```

Comment:
```suggestion
      res.status(204);
```

Since it is our client that is handling the request, we can have the f/e show the message when it gets a 204 response.

Reply:
So for the success case we can do 204, and for the error case we can return a JSON object containing the error message and code?